README.md

```diff
@@ -6,6 +6,16 @@
 
 ## Prerequisites
 
+Note that this README is targeted at AOS-Azure contributors. If you are not a
+member of this team, these instructions may not work as they will assume you
+have permissions that you may not have.
+
+1. **Subscription**. In order to use the OpenShift on Azure plugin, your Azure
```

Comment:
The comments about whitelisting in this paragraph are not really true.  I'd remove the paragraph.

Reply:
Done.